packages/paste-theme-designer/components/topbar/SiteHeaderLogo.tsx

```diff
@@ -34,20 +40,23 @@ const SiteHeaderLogo = ({title, subtitle}) => {
         </MediaFigure>
```

Comment:
The Paste logo feels a little bare not being in a box. Can we use the red background one instead?

Reply:
It's the one we used to have on the docsite before the pride logo I believe.

Reply:
I thought it used to be a white logo in a red square box ? I could be remembering wrong